docs/content/manual/release-specific/v1.1.1/system-upgrade-with-deprecated-cpu-setting.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Test system upgrade with the deprecated CPU setting
+---
+
+## Related issue
+https://github.com/longhorn/longhorn/issues/2207
+
+## Test step
+1. Deploy a cluster that each node has different CPUs.
+2. Launch Longhorn v1.1.0.
+3. Deploy some workloads using Longhorn volumes.
+4. Upgrade to the latest Longhorn version. Validate the workloads work fine and no instance manager pod crash during the upgrade.
```

Comment:
Will the existing value of `Guaranteed Engine Manager CPU` become the new value of `node.Spec.EngineManagerCPURequest` for all the nodes and `Guaranteed Replica Manager CPU` set to a default value? What would be the default value?

Reply:
`node.Spec.EngineManagerCPURequest` is a field in node spec, which is different from the setting `Guaranteed Engine Manager CPU`:
1. The setting will be ignored by the specific node only when its spec field `EngineManagerCPURequest` is not 0.
2. The setting is a percentage value. While the node field holds a milli value.

Reply:
For more details, you can check the doc PR or the LEP PR.